### DISCUSSION TITLE
Spin up for running Dynamic vegetation in Noah-MP within LIS

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1546] [Q&A] [answered]
Hello, I was running Noah-MP version 4.0.1 within NASA LIS version 7.4. I enabled the dynamic vegetation option to check GPP estimates. To get equilibrium carbon balance condition, how long spin up runs are recommended?

**Comment by dmocko** [comment, marked as answer]
Hi @Arijit22IITDelhi 

This question might be better asked on the Noah-MP community discussion pages:
https://github.com/NCAR/noahmp/discussions

In some earlier simulations with Noah-MP-3.6 in LIS, I tried to estimate how long the model would take to reach equilibrium
for several different state variables in the NLDAS-2 domain.  Please see slides 9 through 16 in this presentation:
https://ldas.gsfc.nasa.gov/sites/default/files/ldas/nldas/presentations/Mocko_LIS-NLDAS-update_2015-09-16.pdf

The RootMass state variable only had about 50% of all points reach equilibrium after ~30+ years of simulation.  From what I recall,
this state variable was the slowest to reach equilibrium of the different Mass variables for the prognostic phenology scheme.  I did
not examine the GPP estimates themselves.
